specify/specify7
# fix(build): make Node.js scripts work with Node.js 20

- [x] Don't hardcode Node.js version number in the GitHub Action Workflows, but get it dynamically from the engines field in the package.json
- [x] Update GitHub Action to newest versions
- [x] Run prettier on the .yml files

### Checklist

      and self-explanatory (or properly documented)

### Testing instructions

Most changes in this PR are build-related, so don't affect the user-facing features, as long as everything builds correctly. Still, to be safe, please verify that map in query builder works correctly:

- [ ] Leaflet map shows up correctly in the following places:
  - [ ] When entering Lat/long in query builder, can open the map and specify the coordinates using the ap
  - [ ] If query includes locality table, can plot results on the map
- [x] All GitHub Actions should pass, and use the correct version:
 ![Screenshot 2024-02-19 at 10 29 24](https://github.com/specify/specify7/assets/40512816/d43a4b83-85d4-448d-94af-ac6bc080b81d)
